.github/workflows/cypress.yml

```diff
@@ -1,42 +1,18 @@
 name: Cypress Tests
-on:
-  workflow_call:
-    secrets:
-      CYPRESS_TEST_USER_PASS:
-        required: true
+on: [deployment_status]
+
 jobs:
   Setup:
+    if: github.event.deployment_status.state == 'success'
     runs-on: ubuntu-latest
     steps:
-      - uses: jwalton/gh-find-current-pr@v1
-        id: findPr
-      - name: Set URL for Staging
-        if: github.ref_name == 'main'
-        run: echo "BASE_URL=https://tn-spa-bootstrapper-staging.herokuapp.com/" >> $GITHUB_ENV
-      - name: Set URL for PR
-        if: success() && steps.findPr.outputs.number
-        run: echo "BASE_URL=https://tn-spa-bootstrapper-pr-${PR}.herokuapp.com/" >> $GITHUB_ENV
-        env:
-          PR: ${{ steps.findPr.outputs.pr }}
-      - name: Raise for error
-        if: env.BASE_URL == ''
-        run: exit 1
-      - name: Raise for maintenance mode
-        run: |
-          html_body=$(curl ${{ env.BASE_URL }})
-          if echo $html_body | grep -q "Offline for Maintenance";
-          then
-            exit 1
-          else
-            echo "NOT in maintenance. Likely Heroku trigger after successful deploy";
-          fi
-      - name: Checkout
-        uses: actions/checkout@v2
+      - uses: actions/checkout@v2
       - run: pipx install pipenv
       - uses: actions/setup-python@v2
         with:
           python-version: '3.10'
-          cache: 'pipenv'
+          # Can't use cache because of https://github.com/actions/cache/issues/319
+          # cache: 'pipenv'
```

Comment:
caching doesn't work for `deployment_status` events sadly. (known bug)

Reply:
Is this a big deal at the moment ? No right our apps arent that big that it would mean a huge slow down?

Reply:
It's not really a big deal but it will add time to the test running. So if you push one final commit to the bootstrapper you'll need to wait an extra minute or so.

This does NOT impact the generated projects so definitely not a huge deal in my mind.

Reply:
ahhh okay good call out!